### PR TITLE
Add 'ERROR' status to GCE provisioner

### PR DESCRIPTION
**Pull request by katyafervent**
* Add forgotten ERROR status
* Add provided by google API error description to the cluster metadata

### CODE REVIEW

File: kqueen/engines/gce.py [hunk 1]
```diff
@@ -429,7 +429,7 @@ def cluster_get(self):
             'name': self.cluster_id,
             'id': self.cluster.id,
             'state': state,
-            'metadata': {}
+            'metadata': {'status_message': response['statusMessage']} if 'statusMessage' in response else {}
```
[the remark this answers is not in the document]
Comment (katyafervent) [reply kqueen/engines/gce.py]: Maybe you'll find better or ask me before disapproving the PR? https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.zones.clusters 
'statusMessage' is in output only

[the remark this answers is not in the document]
Comment (naumvd95) [reply kqueen/engines/gce.py]: ill request changes just to freeze PR) ok thanks for clarifying

**Comment by naumvd95** [comment]
also tests are failed for that PR

File: kqueen/engines/gce.py (hunk 1 above)
Comment (ekhomyakova) [inline kqueen/engines/gce.py]: `response['statusMessage']} if 'statusMessage' in response else {}`
->
`response.get('statusMessage', {})`

File: kqueen/models.py [hunk 2]
```diff
@@ -62,6 +64,13 @@ def get_state(self):
 
         return self.state
 
+    def set_status(self, cluster):
+        if cluster:
```
Comment (ekhomyakova) [inline kqueen/models.py]: What do you expect to get that will not pass this condition? Manually passed None or empty string?

Comment (katyafervent) [reply kqueen/models.py]: in case of fail cluster is empty

File: kqueen/engines/gce.py (hunk 1 above)
Comment (katyafervent) [reply kqueen/engines/gce.py]: No, I need empty dictionary or dictionary with the 'status_message' key

File: kqueen/models.py (hunk 2 above)
Comment (ekhomyakova) [reply kqueen/models.py]: Empty dict? If so, you don't have to check it here, because you check it below

File: kqueen/engines/gce.py (hunk 1 above)
Comment (ekhomyakova) [reply kqueen/engines/gce.py]: Oh, didn't notice, okay

**Pull request merged**